packages/flet_flashlight/lib/src/create_control.dart

```diff
@@ -0,0 +1,20 @@
+import "package:flet/flet.dart";
+
+import 'flash.dart';
```

Comment:
Can you rename file to `flashlight.dart`?

Reply:
and this

Reply:
Done.